src/main/java/kitchenpos/application/OrderService.java

```diff
@@ -40,71 +40,84 @@ public OrderService(
     @Transactional
     public Order create(final Order order) {
         final List<OrderLineItem> orderLineItems = order.getOrderLineItems();
+        validateOrderLineItems(orderLineItems);
+        validateMenu(orderLineItems);
+        final OrderTable orderTable = searchOrderTable(order);
+        validateOrderTable(orderTable);
+        Order orderForSave =
+                Order.ofNullId(orderTable.getId(), OrderStatus.COOKING.name(), LocalDateTime.now(), List.of());
+        final Order savedOrder = orderDao.save(orderForSave);
+        fillOrderIdToOrderLineItems(orderLineItems, savedOrder);
+        return savedOrder;
+    }
 
+    private void validateOrderLineItems(final List<OrderLineItem> orderLineItems) {
         if (CollectionUtils.isEmpty(orderLineItems)) {
-            throw new IllegalArgumentException();
+            throw new IllegalArgumentException("[ERROR] 주문 목록이 존재하지 않습니다.");
+        }
+    }
+
+    private void validateMenu(final List<OrderLineItem> orderLineItems) {
+        if (orderLineItems.size() != menuDao.countByIdIn(getMenuIds(orderLineItems))) {
+            throw new IllegalArgumentException("[ERROR] 메뉴가 존재하지 않습니다.");
         }
+    }
 
+    private List<Long> getMenuIds(final List<OrderLineItem> orderLineItems) {
         final List<Long> menuIds = orderLineItems.stream()
                 .map(OrderLineItem::getMenuId)
                 .collect(Collectors.toList());
+        return menuIds;
+    }
 
-        if (orderLineItems.size() != menuDao.countByIdIn(menuIds)) {
-            throw new IllegalArgumentException();
-        }
-
-        order.setId(null);
-
-        final OrderTable orderTable = orderTableDao.findById(order.getOrderTableId())
-                .orElseThrow(IllegalArgumentException::new);
+    private OrderTable searchOrderTable(final Order order) {
+        return orderTableDao.findById(order.getOrderTableId())
+                .orElseThrow(() -> new IllegalArgumentException("[ERROR] OrderTable 을 찾을 수 없습니다."));
```

Comment:
요 에러 메세지도 도메인 용어를 사용하면 좋을 것 같아요 😊
```suggestion
                .orElseThrow(() -> new IllegalArgumentException("[ERROR] 테이블을 찾을 수 없습니다."));
```

Reply:
그러네요 제가 놓친거 같습니다. 관련해서 에러 메시지들을 통일했습니다!